package.json

```diff
@@ -37,7 +37,18 @@
 
     "serve": "ws --directory ./client/dist --spa ./tools/index.html --port 3001 --rewrite '/ -> /tools'",
     "serve-forever": "forever start -l forever.log -o logs/out.log -e logs/error.log -a -p logs --minUptime 1000 --spinSleepTime 1000 ./node_modules/.bin/ws --directory ./client/dist --spa ./tools/index.html --port 3001 --rewrite '/ -> /tools'",
-    "start": "npm run dev"
+    "start": "npm run dev",
+
+
+    "lint:eslint": "./node_modules/.bin/eslint --ext js --fix . .config",
+    "pretest": "npm run update-webdriver && npm run selenium-server-standalone",
+    "test": "npm run e2e-local",
```

Comment:
There is no karma configuration for this project jet.
Should i configure some defaults or should skip 'npm test' section?